.github/workflows/test.yml

```diff
@@ -1015,3 +1016,57 @@ jobs:
       - name: valgrind
         run: |
           nix develop -c zig build test-valgrind
+
+  build-freebsd:
+    name: Build on FreeBSD
+    needs: test
+    runs-on: namespace-profile-mitchellh-sm-systemd
+    strategy:
+      matrix:
+        release:
+          - "14.3"
+          # - "15.0" # disable until fixed: https://github.com/vmactions/freebsd-vm/issues/108
+    steps:
+      - name: Checkout Ghostty
+        uses: actions/checkout@08eba0b27e820071cde6df949e0beb9ba4906955 # v4.3.0
```

Comment:
Why not use the tag name there?

Reply:
We hardcode to hashes throughout so that the upgrades of every individual commit are controlled by us, so we aren't subject to the various GHA attacks that have happened multiple times before.